requirements/dist-build-constraints.in

```diff
@@ -5,7 +5,6 @@
 setuptools >= 34.4
 
 # Plugins
-setuptools_scm[toml] >= 3.5
```

Comment:
This must not be removed as it cascades into the transitive constraints not being present in the constraint file.